handbook/marketing/oss_community_pages.md

```diff
@@ -1,12 +1,27 @@
-
-For now, those pages are based on repogroups (a named group of repositories defined by a site admin on Sourcegraph.com).
+Sourcegraph hosts community landing pages to help open source community members discover, onboard, and search their code. Good examples include the [kubernetes](https://sourcegraph.com/kubernetes) and [stanford](https://sourcegraph.com/stanford) landing pages. The pages are based on search contexts (a named collection of repositories and revisions).
 
 ## How to create a community landing page on Sourcegraph Cloud
 
-- Identify the requirements for the repogroup page. What repos should be included in the repogroup? What examples would be worth highlighting?
-- Create the repogroup by opening a PR adding a repogroup definition to `search.repositoryGroups` [here](https://github.com/sourcegraph/deploy-sourcegraph-dot-com/blob/release/base/frontend/sourcegraph-frontend.ConfigMap.yaml#L47). Tag search-product for review.
-- Create the page ([example](https://github.com/sourcegraph/sourcegraph/commit/576318e4dff2a3ecc8002ebea2b490b8ee99fc31)) and open a PR. Tag search-product for review.
-- Run a few tests to check that repositories are all indexed and the repo group returns expected results.
+First, create the search context:
+
+- Make sure you are a site-admin on Cloud
+- Identify the requirements for the community search contexts page. What repos should be included in the search context? What examples would be worth highlighting?
+- Create a new search context: [Create page](https://sourcegraph.com/contexts/new)
+  - Select the "Global owner" option from the Owner dropdown (this will make the context available to all users on Cloud)
+  - Enter the context name (e.g. `stanford`, `cncf`)
+  - Enter a description (Markdown is supported)
+  - Select the "Public" visiblity
+  - Enter the repositories as a JSON config (see [Stanford edit page](https://sourcegraph.com/contexts/stanford/edit) for example configuration)
+  - Click "Test configuration" to check if the configuration is in the correct format and that all of the entered repositories exist
+  - Finally, you can create the search context
+
+Second, create the corresponding community search context page in the main sourcegraph repository. This is a temporary measure and will be automated in the future.
+Steps:
+
+- Copy an existing page config (e.g. [Stanford](https://sourcegraph.com/github.com/sourcegraph/sourcegraph/-/blob/client/web/src/communitySearchContexts/Stanford.tsx)) and adjust the necessary data
+  - **Important**: The `spec` property should match the search context name you created in the first step
+- Add the corresponding route [to the client router](https://sourcegraph.com/github.com/sourcegraph/sourcegraph/-/blob/client/web/src/communitySearchContexts/routes.tsx?L31)
```

Comment:
L31 is not expected to always point to the same thing. Could we make these links pin a commit so they'll at least always make sense?

Reply:
Good point, I'll update all of the links.